pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -38,14 +39,29 @@ type webhookOptions struct {
 type webhookClient interface {
 	Create(ctx context.Context, options *webhookOptions) (string, error)
```

Comment:
Are `Create` and `Delete` still used? If not, remove them.

Reply:
Ah ... yes, I hadn't noticed the lowercase and hadn't wanted to cause version issues for people using the interface.  But as its's not exported I've modified code back to having Create and Delete methods with the extra arg (rather than having renamed methods)